### ISSUE TITLE
2 configs in Debian package

### ISSUE THREAD

**Issue by szepeviktor** [allinurl/goaccess#1364]
Could it be that the Debian package from deb.goaccess.io contains 2 config files?

```
$ dpkg -S goaccess.conf
goaccess: /etc/goaccess/goaccess.conf
goaccess: /etc/goaccess.conf
```

**Comment by roben** [comment]
This happened for me after changing from the debian repo to the official one. The official repo creates `/etc/goaccess/goaccess.conf` but not `/etc/goaccess.conf`. Nevertheless, the binary tries to load `/etc/goaccess.conf` and exits with an error if it does not exist. I have symlinked the file for now to fix this.

**Comment by allinurl** [comment]
I believe this was fixed in #1395 please feel free to build from [**development**](https://github.com/allinurl/goaccess#build-from-github-development) and let me know if that resolve the issue for you. Thanks!

**Comment by allinurl** [comment]
This has been fixed and it's now available in [**v1.4**](https://goaccess.io/download). Thank you!
